.travis/before_install.sh

```diff
@@ -116,7 +116,8 @@ cd /tmp || die
         cd wpantund || die
         ./bootstrap.sh || die
         ./configure || die
-        sudo make install -j 8 || die
+        sudo make -j 8 || die
```

Comment:
I guess technically this line doesn't need to be sudo, but I guess it doesn't really matter from the context of travis.